pyodbc.cpython-38-aarch64-linux-gnu.so,sha256=1b9uXUnCGchpPAZrwtH1nwbMcrSpu99_N17l6h_VIPo,923689
pyodbc-5.0.0a2.dist-info/RECORD,,
pyodbc-5.0.0a2.dist-info/WHEEL,sha256=0hhP97AV4kXP0ejB-F3ZqHePUSlgfnH2eMaBqXSBAKQ,150
pyodbc-5.0.0a2.dist-info/LICENSE.txt,sha256=mFnIV3DJG3hmsPdpyTjjbWTQJgFBtSm5R7Tq2WiD6rQ,868
pyodbc-5.0.0a2.dist-info/METADATA,sha256=rmKkah_waYVn7-pIvd_Nu8DsAeuAQxTTAVQpJ9DZKlw,2431
pyodbc-5.0.0a2.dist-info/top_level.txt,sha256=30jvYcCul6_wA26OT5c068GAHzqQLpd9WfQ0mSSsDTU,7
pyodbc-5.0.0a2.data/data/pyodbc.pyi,sha256=JWNpdgmrxhNioTjW6OoAEcXELcGZobcMYzKtYlcJRuY,32320
